### PR TITLE
Merge Request  "Localization Issue #258" - Spanish Translation

### PR DESCRIPTION
**Pull request by 4troDev**
# Reference issue

-   Localization #258 

## Purpose

-   `The purpose of this merge request is to address issue #258, which focuses on Localization. Specifically, a new language file for Spanish localization, 'es.ts', has been added to translate the authentication module ('authme') into Spanish.`

## Approach

-   `To address the issue, a new language file, 'interface/utils/languages/es.ts', has been introduced. 

## TODO

-   [x] Read [Contributing](https://github.com/Levminer/authme/blob/main/.github/CONTRIBUTING.md)
-   [x] Read [Code of conduct](https://github.com/Levminer/authme/blob/main/.github/CODE_OF_CONDUCT.md)
-   [x] Run `npm run lint`

### CODE REVIEW

File: interface/utils/language/es.ts [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+		clearDataText: "Borre la contraseña, los códigos 2FA y todas las demás configuraciones. ¡Cuidado, esto no se puede deshacer!",
+		clearDataButton: "Borrar datos",
+		codes: "Codigos",
+		codesDescription: "Descripción de códigosn",
+		codesDescriptionText: "La descripción de los códigos 2FA será visible. Puedes copiarlo después de hacer clic en él.",
+		blurCodes: "Códigos de desenfoque",
+		blurCodesText: "Difumina los códigos guardados. Aún puedes copiar los códigos o pasar el cursor sobre los códigos para mostrarlos.",
+		codesLayout: "Diseño de códigos",
+		codesLayoutText: "Puede elegir su diseño preferido. Grid muestra más elementos y se adapta al tamaño de la pantalla.",
+		sortCodes: "ordenar códigos",
```
Comment (Levminer) [inline interface/utils/language/es.ts]: Capitalize the first letter

File: interface/utils/language/es.ts [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+
+	settings: {
+		general: "General",
+		launchOnStartup: "Lanzar al inicio",
+		launchOnStartupText: "Inicie Authme automáticamente al iniciar el sistema en la bandeja del sistema.",
+		minimizeToTray: "Minimizar a la bandeja",
+		minimizeToTrayText: "Al cerrar la aplicación, Authme no se cerrará. Puede abrir Authme desde la bandeja del sistema.",
+		optionalAnalytics: "Análisis opcionales",
+		optionalAnalyticsText: "La carga útil enviada es completamente anónima. Los datos de envío incluyen su versión de Authme e información sobre su computadora.",
+		windowCapture: "captura de ventana",
```
Comment (Levminer) [inline interface/utils/language/es.ts]: Capitalize the first letter

File: interface/utils/language/es.ts [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+		noAuthText: "Si no desea escribir una contraseña cada vez que inicia Authme.",
+		hardwareAuth: "Autenticación de hardware",
+		hardwareAuthText: "Confirme acciones importantes con Windows Hello, Touch ID o cualquier clave de hardware compatible con WebAuthn.",
+		createPass: "Crear contraseña",
+		createPassText: "Cree una contraseña segura para cifrar sus códigos.",
+		password: "Contraseña",
+		confirmPassword: "Confirmar Contraseña",
+		chooseDifferent: "Elija una autenticación diferente",
+		chooseDifferentText: "Vuelva a la página anterior y elija un método de autenticación diferente.",
+		goBack: "regresar",
```
Comment (Levminer) [inline interface/utils/language/es.ts]: Capitalize the first letter

**Review by Levminer — CHANGES_REQUESTED** [review]
Thanks four your work! Please address the issues I requested.

File: interface/utils/language/es.ts [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+		authmeFile: "Archivo de Authme",
+		authmeFileText: "Importe todos los códigos de un archivo de importación de Authme existente.",
+		authmeFileButton: "Elija el archivo",
+		// html dialogs
+		captureScreenTitle: "Importación de pantalla de captura",
+		captureScreenWaiting: "Esperando un código QR...",
+		manualEntry: "Configurar una clave",
+		manualEntryText: "Ingrese un nombre y una clave secreta 2FA.",
+		manualEntryName: "Nombre (requerido)",
+		manualEntrySecret: "Secreto (obligatorio)",
```
Comment (Levminer) [inline interface/utils/language/es.ts]: Whats the difference between requerido and obligatorio? You should only use one

**Comment by 4troDev** [comment]
I've updated the issues

File: interface/utils/language/es.ts (hunk 4 above)
Comment (4troDev) [reply interface/utils/language/es.ts]: Depending on the situation, you may prefer "requerido" (which means required) over "obligatorio" (which means obligatory).

**Review by Levminer — APPROVED** [review]

**Pull request merged**